packages/kmod-6.1-nvidia/nvidia-open-gpu-load-config.toml.in

```diff
@@ -0,0 +1,16 @@
+[nvidia-open-gpu]
+lib-modules-path = "kernel/drivers/extra/video/nvidia/open-gpu"
+objects-source = "__NVIDIA_MODULES__"
+load-only = true
```

Comment:
Can load-only be inferred from the fact that all of the link objects lists are empty?

That seems less error-prone.